specs/_features/whisk/fork.md

```diff
@@ -124,12 +115,12 @@ def upgrade_to_whisk(pre: bellatrix.BeaconState) -> BeaconState:
 
     # Do a candidate selection followed by a proposer selection so that we have proposers for the upcoming day
     # Use an old epoch when selecting candidates so that we don't get the same seed as in the next candidate selection
-    whisk_candidate_selection(post, epoch - WHISK_PROPOSER_SELECTION_GAP - 1)
-    whisk_proposer_selection(post, epoch)
+    select_whisk_candidate_trackers(post, max(0, epoch - WHISK_PROPOSER_SELECTION_GAP - 1))
```

Comment:
`remerkeleable` would auto-cast the result into SSZ `uint64` here and will raise an underflow error when we assign a negative uint64. The full pyspec-compatible solution would be:

```python
result_epoch = epoch - WHISK_PROPOSER_SELECTION_GAP - 1 if epoch > WHISK_PROPOSER_SELECTION_GAP + 1 > 0 else Epoch(0)
select_whisk_candidate_trackers(post, result_epoch)
```

It may hurt readability more though. 😅

Reply:
I like this helper as well